types/fhir/index.d.ts

```diff
@@ -61,6 +61,1049 @@ declare module fhir {
      * An integer with a value that is not negative (e.g. >= 0)
      */
     type unsignedInt = number;
+    /**
+     * This value set includes Status codes
+     */
+    type FinancialResourceStatusCode = 'active' | 'cancelled' | 'draft' | 'entered-in-error';
+    /**
+     * The use of an address
+     */
+    type AdressUse = 'home' | 'work' | 'temp' | 'old';
+    /**
+     * The type of an address (physical / postal)
+     */
+    type AdressType = 'postal' | 'physical' | 'both';
+    /**
+     * How the Quantity should be understood and represented
+     */
+    type QuantityComparator = '<' | '<=' | '>=' | '>';
+    /**
+     * Identifies the purpose for this identifier, if known
+     */
+    type IdentifierUse = 'usual' | 'official' | 'temp' | 'secondary';
+    /**
+     * Telecommunications form for contact point
+     */
+    type ContactPointSystem = 'phone' | 'fax' | 'email' | 'pager' | 'url' | 'sms' | 'other';
+    /**
+     * Use of contact point
+     */
+    type ContactPointUse = 'home' | 'work' | 'temp' | 'old' | 'mobile';
+    /**
+     * The use of a human name
+     */
+    type NameUse = 'usual' | 'official' | 'temp' | 'nickname' | 'anonymous' | 'old' | 'maiden';
+    /**
+     * A unit of time (units from UCUM)
+     */
+    type UnitsOfTime = 's' | 'min' | 'h' | 'd' | 'wk' | 'mo' | 'a';
+    /**
+     * The status of a resource narrative
+     */
+    type NarrativeStatus = 'generated' | 'extensions' | 'additional' | 'empty';
+    /**
+     * Indicates whether the account is available to be used
+     */
+    type AccountStatus = 'active' | 'inactive' | 'entered-in-error';
+    /**
+     * The type of contributor
+     */
+    type ContributorType = 'author' | 'editor' | 'reviewer' | 'endorser';
+    /**
+     * The type of relationship to the related artifact
+     */
+    type RelatedArtifactType =
+        | 'documentation'
+        | 'justification'
+        | 'citation'
+        | 'predecessor'
+        | 'successor'
+        | 'derived-from'
+        | 'depends-on'
+        | 'composed-of';
+    /**
+     * The type of participant for the action
+     */
+    type ActionParticipantType = 'patient' | 'practitioner' | 'related-person';
+    /**
+     * The lifecycle status of a Value Set or Concept Map
+     */
+    type PublicationStatus = 'draft' | 'active' | 'retired' | 'unknown';
+    /**
+     * TODO
```

Comment:
This is not my comment - it is the actual comment in the definition: http://www.hl7.org/fhir/STU3/codesystem-adverse-event-causality.html

Reply:
Maybe just leave it uncommented, or add `@see http://www.hl7.org/fhir/STU3/codesystem-adverse-event-causality.html` instead? A year from now someone might look at this and file a bug because of the TODO.